src/data/markdown/docs/01 guides/05 Testing Guides/04 Running large tests.md

```diff
@@ -485,8 +478,65 @@ Results
 Note: each VU in k6 is completely independent, and therefore it doesn't share any memory with other VUs.
 1000VUs uploading 26MB file need as much as 81GB of RAM since each VU holds the copy of the file in memory.
 
+## Distributed execution
+
+In load testing, distributed execution refers to running a load test distributed across multiple machines. 
+
+Users often look for the distributed execution mode to run large-scale tests. Although we have shown that a single k6 instance can generate enormous load, distributed execution is necessary to:
+
+- Simulate load from multiple locations simultaneously.
+- Scale the load of your test beyond what a single machine can handle.
+
+In k6, you can split the load of a test across multiple k6 instances using the [execution-segment](/using-k6/options#execution-segment) option. For example:
+
+
+<div class="code-group" data-props='{"labels": ["Two machines", "Three machines", "Four machines"]}'>
+
+```bash
+## split the load of my-script.js across two machines
+k6 run --execution-segment "0:1/2"   --execution-segment-sequence "0,1/2,1" my-script.js
+k6 run --execution-segment "1/2:1"   --execution-segment-sequence "0,1/2,1" my-script.js
+```
+
+```bash
+## split the load of my-script.js across three machines
+k6 run --execution-segment "0:1/3"   --execution-segment-sequence "0,1/3,2/3,1" my-script.js
+k6 run --execution-segment "1/3:2/3" --execution-segment-sequence "0,1/3,2/3,1" my-script.js
+k6 run --execution-segment "2/3:1"   --execution-segment-sequence "0,1/3,2/3,1" my-script.js
+```
+
+```bash
+## split the load of my-script.js across four machines
+k6 run --execution-segment "0:1/4"     --execution-segment-sequence "0,1/4,2/4,3/4,1" my-script.js
+k6 run --execution-segment "1/4:2/4"   --execution-segment-sequence "0,1/4,2/4,3/4,1" my-script.js
+k6 run --execution-segment "2/4:3/4"   --execution-segment-sequence "0,1/4,2/4,3/4,1" my-script.js
+k6 run --execution-segment "3/4:1"     --execution-segment-sequence "0,1/4,2/4,3/4,1" my-script.js
+```
+
+</div>
+
+However - at this moment - the distributed execution mode of k6 is not entirely functional. The current limitations are:
+
+- k6 does not provide a `test coordinator` or `master instance` to coordinate the distributed execution of the test. Alternatively, you can use the [k6 REST API](/misc/k6-rest-api) and [--paused](/using-k6/options#paused) to synchronize the multiple k6 instances' execution. 
+- Each k6 instance evaluates [Thresholds](/using-k6/thresholds) independently - excluding the results of the other k6 instances. If you want to disable the threshold execution, use [--no-thresholds](/using-k6/options#no-thresholds).
+- k6 reports individually the metrics for each instance. Depending on how you store the load test results, you'll have to aggregate some metrics to calculate them correctly. 
```

Comment:
grammar nitpick. Change "k6 reports individually the metrics " to " k6 reports the metrics individually"

Reply:
https://github.com/loadimpact/k6-docs/pull/102/commits/d602a8d75d00320b4adc4a4f5b5e4b1c77e2daa5